test/mongodb.test.js

```diff
@@ -1032,6 +1032,16 @@ describe('mongodb connector', function() {
   });
 
   describe('updateAll', function() {
+    it('should preserve the id', async function() {
+      let user = await User.create({name: 'Al', age: 31, email: 'al@strongloop'});
+      const userId = user.id;
+      user = user.toObject();
```

Comment:
Let's call this property `userData` to make it clear that it's a plain-data object, not a model instance. (And also to avoid a mutable variable `user`.)

```suggestion
      userData = user.toObject();
```

Reply:
I am afraid my comment was not resolved :(